src/components/content/Content.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import ReactHtmlParser from 'react-html-parser'; 
+
+export interface ContentProps {
```

Comment:
I was curious about this stuff - I am guessing this is something typescript related...? I'm getting red squiggly lines under it in VS code but don't want to take it out if it's something you are using!